Archives/ClmFile.h

```diff
@@ -71,7 +71,7 @@ namespace Archives
 		void PackFile(StreamWriter& clmWriter, const IndexEntry& entry, StreamReader& fileToPackReader);
 		std::vector<std::string> StripFileNameExtensions(std::vector<std::string> paths);
 		void InitializeWaveHeader(WaveHeader& headerOut, int fileIndex);
-		WaveFormatEx CreateDefaultWaveFormat();
+		static WaveFormatEx CreateDefaultWaveFormat();
```

Comment:
Ahh. I see the miscommunication now. I meant the data structure itself could be a static class member. I suppose it makes sense for the function returning the data to be static, though I was thinking in terms of eliminating the function and just accessing the static const struct directly.

Granted my idea might not be quite perfect, in that it's questionable if the default format should be part of the class (representing the file format), or part of the front-end (representing what Outpost 2 expects for its use of this file format). Perhaps a decision for another day.

At any rate, your current solution is correct code. I'm just trying to clarify my original intention in the first comment. Any changes here are entirely optional.